app/controllers/items_controller.py

```diff
@@ -0,0 +1,230 @@
+    @api.response(401, 'Unauthorized')
+    @api.response(403, 'Forbidden operation')
+    @api.response(404, 'Not found')
+    @token_required
+    def delete(self, item_id: int):
+        """Delete item in own storage by id"""
+        user = get_user_from_request(api)
+        item = Items.get_item_by_id(item_id)
+        if item is not None:
+            storage = Storages.get_storage_by_id(item['storage_id'])
+            if user['user_id'] == storage['user_id']:
+                item_response = Items.delete(item_id)
+                if item_response:
+                    return '', 204
+                api.abort(HTTPStatus.FORBIDDEN, f'Item {item_id} can\'t be deleted')
+            api.abort(HTTPStatus.FORBIDDEN, f'Item {item_id} is not yours')
+        api.abort(HTTPStatus.NOT_FOUND, f'Item {item_id} not found')
+
+
+@api.route('/storage/<int:storage_id>')
+@api.doc(security='access-token')
+class ItemsMeByStorageIdApi(Resource):
+    @api.doc('get_items_by_storage')
+    @api.response(200, 'Success', [ItemsModels.item])
+    @api.response(401, 'Unauthorized')
+    @api.response(403, 'Forbidden operation')
+    @api.response(404, 'Not found')
+    @token_required
+    def get(self, storage_id: int):
+        """Get items by storage"""
+        requester = get_user_from_request(api)
+        check_admin(api, requester)
+        storage = Storages.get_storage_by_id(storage_id)
+        if storage is not None:
+            return Items.get_items_by_storage(storage['storage_id'])
+        api.abort(HTTPStatus.NOT_FOUND, f'Storage {storage_id} not found')
+
+
+@api.route('/search')
+class ItemsSearch(Resource):
+    @api.doc('search')
+    @api.expect(ItemsModels.search, validate=True)
+    @api.response(200, 'Search completed', [ItemsModels.item_public])
+    def get(self):
+        """Search"""
+        args = ItemsModels.search.parse_args()
+        content = args.content
+        name_ru = [Items.orm2dict(it) for it in Items.query.filter_by(name_ru=content).order_by(Items.item_id).all()]
+        name_en = [Items.orm2dict(it) for it in Items.query.filter_by(name_en=content).order_by(Items.item_id).all()]
+        desc_ru = [Items.orm2dict(it) for it in Items.query.filter_by(desc_ru=content).order_by(Items.item_id).all()]
+        desc_en = [Items.orm2dict(it) for it in Items.query.filter_by(desc_en=content).order_by(Items.item_id).all()]
+        res_search = name_ru
+        res_search.extend(it for it in name_en if it not in res_search)
+        res_search.extend(it for it in desc_ru if it not in res_search)
+        res_search.extend(it for it in desc_en if it not in res_search)
+        res = []
+        for item in res_search:
+            storage = Storages.get_storage_by_id(item['storage_id'])
+            user = Users.get_user_by_id(storage['user_id'])
+            united = OrderedDict([('owner', user)])
+            united.update(item)
+            res = marshal(united, ItemsModels.item_public)
```

Comment:
Oh, you need to `append` a new value to the list instead of redefining the `res` variable:
```py
            res.append(marshal(united, ItemsModels.item_public))
```